src/main/java/heronarts/lx/color/LXPalette.java

```diff
@@ -528,6 +539,10 @@ private void _transitionColor(LXDynamicColor color, int i, float lerp) {
     }
   }
 
+  private boolean isBlack(ColorParameter color) {
```

Comment:
Let's put this method on `ColorParameter` so it's `color.isBlack()` rather than method living in `LXPalette` and taking a `ColorParameter` argument

Reply:
Good, done.